src/web/lib/hakuneko/frontend@classic-dark/pages.html

```diff
@@ -516,6 +525,42 @@
                     this.$.container.scrollTop = previousOffset * this.$.container.scrollHeight / previousHeight;
                 //}.bind( this ), 0 );
             }
+
+            /**
+             * Dynamically change the scrolling to stop at the end of images or skip to the start of the next image
+             * Usefull for manga reading but ugly with webtoons
+             */
+            scrollMagic(defaultDistance) {
+                let images = this.$.container.getElementsByClassName('image');
+                // Find current image within view
+                let currentScrollImages = Array.prototype.filter.call(images, function(image){
```

Comment:
May use spread operator `[...images].filter(image => { /* fancy stuff here */ })`

Reply:
Oh, seems easier to read indeed. Never used spread op before (quite the old fashion one)